FOSS/content/IdentityModel/endpoints/general_usage.md

```diff
@@ -94,3 +94,49 @@ Some client libraries also include a stateful client object (e.g.
 *TokenClient* and *IntrospectionClient*). See the corresponding section
 to find out more.
 {{% /notice %}}
+
+Client Credential Style
+------------------------
+
+Any request type implementing *ProtocolRequest* has the ability to configure
+the client credential style, which specifies how the client will transmit the client ID and secret.
+*ClientCredentialStyle* options include *PostBody* and the default value of *AuthorizationHeader*.
+
+```cs
+var client = HttpClientFactory.CreateClient("my_named_token_client");
+
+var response = await client.RequestClientCredentialsTokenAsync(
+    new ClientCredentialsTokenRequest
+    {
+        Address = "https://demo.duendesoftware.com/connect/token",
+        ClientId = "client",
+        ClientSecret = "secret",
+        // set the client credential style
+        ClientCredentialStyle = ClientCredentialStyle.AuthorizationHeader
+    });
+```
+
+While both options are functionally equivalent, both have their advantages and disadvantages.
```

Comment:
Let's repharse this: "both" x2 feels awkard, and they're not functionality equivalent - behavior is actually different.